doc/whats_new/v0.22.rst

```diff
@@ -2,6 +2,48 @@
 
 .. currentmodule:: sklearn
 
+.. _changes_0_22_2:
+
+Version 0.22.2
+==============
+
+**February 28 2020**
```

Comment:
I like your optimism :P

Reply:
:smile:

Reply:
![Lack of Faith](https://media.giphy.com/media/M4iOAkEAPwAnK/giphy.gif)